src/Controls/src/Core/Platform/Windows/CollectionView/FormsGridView.cs

```diff
@@ -103,23 +100,18 @@ void OnChoosingItemContainer(ListViewBase sender, ChoosingItemContainerEventArgs
 			FindItemsWrapGrid();
 		}
 
-		void WrapGridSizeChanged(object sender, SizeChangedEventArgs e)
-		{
-			UpdateItemSize();
-		}
-
-		void UpdateItemSize()
+		internal void UpdateItemSize()
 		{
 			// Avoid the ItemWrapGrid grow beyond what this grid view is configured to
 			_wrapGrid.MaximumRowsOrColumns = Span;
```

Comment:
With the changes, I would include a validation if `_wrapGrid` is not null here.

Reply:
@jsuarezruiz, As suggested, I have added the null check condition for _WrapGrid.